pkg/workflows/workflows_test.go

```diff
@@ -63,7 +63,7 @@ func TestCreateWorkflows(t *testing.T) {
 	deployType := "helm"
 	flagVariables := []string{}
 	templatewriter := &writers.LocalFSWriter{}
-	flagValuesMap := map[string]string{"AZURECONTAINERREGISTRY": "testAcr", "CONTAINERNAME": "testContainer", "RESOURCEGROUP": "testRG", "CLUSTERNAME": "testCluster", "BRANCHNAME": "testBranch"}
+	flagValuesMap := map[string]string{"AZURECONTAINERREGISTRY": "testAcr", "CONTAINERNAME": "testContainer", "RESOURCEGROUP": "testRG", "CLUSTERNAME": "testCluster", "BRANCHNAME": "testBranch", "BUILDCONTEXTPATH": "."}
```

Comment:
would it be useful to have a test to ensure that path behavior is exactly what we expect it to be? eg, does it work when we put ./dirname, and/or dirname? feel like having tests for that use case will most closely align with what we can expect users to do since if they have a specific build context, it probably won't just be the root of their repo